### ISSUE TITLE
unable to trigger job

### ISSUE THREAD

**Issue by mengxu2018** [concourse/concourse#4019]
I was new to concourse, and set up the environment in my centos7.6 like below.
```
$ wget https://concourse-ci.org/docker-compose.yml
$ docker-compose up -d
Then login by `fly --target example login --team-name main --concourse-url  http://192.168.77.140:8080/ -u test -p test`
```

I can see below.
```
[root@centostest ~]# fly targets
name     url                         team  expiry
example  http://192.168.77.140:8080  main  Sun, 16 Jun 2019 02:23:48 UTC
```

I used below yaml.xml named with **2.yaml**
```
---
resources:
- name: my-git-repo
  type: git
  source:
    uri: https://github.com/ruanbekker/concourse-test
    branch: basic-helloworld

jobs:
- name: hello-world-job
  public: true
  plan:
  - get: my-git-repo
  - task: task_print-hello-world
    file: my-git-repo/ci/task-hello-world.yml
```
Then I run below commands step by step.
```
fly -t example sp -c 2.yaml -p pipeline-01
fly -t example up -p pipeline-01
fly -t example tj -j pipeline-01/hello-world-job --watch
```
But i just hang on there , no useful response like below.
```
[root@centostest ~]# fly -t example tj -j pipeline-01/hello-world-job --watch
started pipeline-01/hello-world-job #3



```
Theoretically， it should print something like below.
```
Cloning into '/tmp/build/get'...
Fetching HEAD
292c84b change task name
initializing
running echo hello world
hello world
succeeded
```

Where I did wrong? thanks.

**Comment by pivotal-bin-ju** [comment]
Hi mengxu2018,
Could you find some error from the log?
If you start up concourse with the command (without the `-d`)

```
docker-compose up
```
then you can see the log from the console.

Or you could run the command:

```
docker-compose logs web|grep error
```
to see those errors

**Comment by mengxu2018** [comment]
@pivotal-bin-ju 
 At last, I changed to setup the environment manually instead of docker, but still met the hang issue, I resolved it by upgrading the linux kernel to 5.1,  I was using the centos 7.6.
